src/PluginClient.php

```diff
@@ -110,6 +111,7 @@ private function configure(array $options = [])
         $resolver = new OptionsResolver();
         $resolver->setDefaults([
             'max_restarts' => 10,
+            'debug_plugins' => [],
```

Comment:
Can we add a type check for plugins somewhere? Does the optionresolver support "an array of type" check?